serve.go

```diff
@@ -746,3 +807,28 @@ func apiAuthMiddleware(token string, next http.Handler) http.HandlerFunc {
 
 	}
 }
+
+// implement HasCN function that belongs in micromdm/scep/depot/bolt
+//   note: added bool return, different from micromdm/scep interface
```

Comment:
Definitely. Actually I'm not sure about merging this until its implemented in micromdm/scep? What do you think? That way I can toss the bolt.Depot struct reference and instead use a scep.Depot interface in the config struct.